crates/nu-command/src/filesystem/open.rs

```diff
@@ -281,6 +281,16 @@ impl Command for Open {
                 example: r#"def "from ndjson" [] { from json -o }; open myfile.ndjson"#,
                 result: None,
             },
+            Example {
+                description: "Show the extensions for which the `open` command will automatically parse",
+                example: r#"scope commands
+    | where name starts-with "from "
+    | insert extension { get name | str replace -r "^from " "" | $"*.($in)" }
+    | select extension name
+    | rename --column { name: command }
```

Comment:
A minor simplification would be
```suggestion
    | rename extension command
```